src/routes/SecuredRoutes.jsx

```diff
@@ -9,13 +9,25 @@ import { Login } from '../components';
 import { jwtInterceptor } from './interceptors';
 import { OKTA_AUTH_CONFIG, SENTRY_CONFIG } from './config';
 
-const Home = lazy(() => import('../pages/home/Home'));
+
+const FeaturedMovies = lazy(()=> 
+    import('../pages/featuredMovies'));
+const Home = lazy(() => 
+    import('../pages/home/Home'));
```

Comment:
the last '/Home' can be removed here once we add default exports in home/index

Reply:
closing this PR considering the time constraints, will cover this point in future branches